HW/feature_eng.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+# user input to receive dataset
+# https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data
+dataset = pd.read_csv(
+    "https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data"
```

Comment:
I was having the weirdest time figuring out why your code was taking > 5 minutes to run and it turns out this line was basically the whole wait. When I went to this link in my browser it said the download rate was less than 500 B/s. Maybe the site is just lagging right now or something. But it doesn't look like a big file so you can speed this up by opening it from a local file instead. (or maybe I just hit the site at a bad time?)